dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/attribute/DefaultAttributeService.java

```diff
@@ -187,7 +184,7 @@ public Attribute getAttributeByCode( String code )
         }
 
         object.getAttributeValues().add( attributeValue );
-        sessionFactory.getCurrentSession().save( object );
+        manager.update( object );
```

Comment:
Like methods removing attribute values the modified object is updated, not created.